system_reports/osx-xcode-9.4.x.log

```diff
@@ -1,4 +1,4 @@
- (i) Ignore Errors: disabled
+ (i) Ignore Errors: enabled
```

Comment:
Is this ok?

Reply:
Yes, for older stacks mainly, to document which tools are *not* installed on those stacks.